dist/general.go

```diff
@@ -11,15 +11,15 @@ type Parameter struct {
 // ParameterMarshaler exists to support algorithms that modify parameters of arbitrary distributions.
 // Typically, users should modify distributions using the fields of the specifc
 // distribution.
-// Marshal and Unmarshal are to be used as a pair, users should not attempt to construct
```

Comment:
Could we rephrase this rather than remove it? I agree the wording is weird, but the warning is useful. I would suggest something like "users should not attempt to construct the input to UnmarshalParameters themselves"